### DISCUSSION TITLE
Using the collection orderable function with runway

### DISCUSSION THREAD

**Discussion by jornwildenbeest** [statamic-rad-pack/runway#171] [Help]
Hi!

Is possible to use the collection orderable function in the cp?

For example, using the order function from runway: https://runway.duncanmcclean.com/resources#content-ordering
But with a column or something that is filled by the order using the orderable function in the cp?

**Comment by duncanmcclean** [comment]
Thanks for opening a discussion!

Unfortunately, I'm not sure I quite understand what you're asking... I don't suppose you'd be able to record [a short screen recording](https://zipmessage.com/nitcffb8) showing what you mean?

**Reply by duncanmcclean** [reply]
In fact, is this the kind of thing you mean? (because it should already be possible)

https://zipmessage.com/nitcffb8/156a856ad9e6a967266363afb

**Reply by jornwildenbeest** [reply]
Sorry for the late response. I am talking about the following Statamic feature: 

![image](https://user-images.githubusercontent.com/25662412/195036366-6e4e761c-860a-4c2d-be72-ed59f6ee3b31.png)

Once clicking, you will see this drag and drop icon:

![image](https://user-images.githubusercontent.com/25662412/195036555-0c363ff2-505f-49d1-ab84-22ae4daedfd8.png)

With this you, using drag and dop, change the order in statamic. Don't think this in runway yet because i didn't read about this in the docs, but maybe this would be a nice new feautre?

**Reply by duncanmcclean** [reply]
Ah yes, that's not something that's in Runway. I didn't know that was even a feature with collections (outside structured collections).

I'm not sure how many people would benefit from it - if there's enough people who want it, I'm happy to build it. I'd also accept a PR.
